src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -1750,7 +1750,7 @@ private Symbol GetDeclaredMember(NamespaceOrTypeSymbol container, TextSpan decla
         /// <summary>
         /// Finds the member in the containing symbol which is inside the given declaration span.
         /// </summary>
-        private Symbol GetDeclaredMember(NamespaceOrTypeSymbol container, TextSpan declarationSpan, string name = null)
+        private Symbol GetDeclaredMember(NamespaceOrTypeSymbol container, TextSpan declarationSpan, bool isKnownNamespace, string name = null)
```

Comment:
I am not clear what `isKnownNamespace` indicates. Can you please add details to the doc comment?

Reply:
➡️ Amended the commit to include a comment. The diff for this can be reviewed in https://github.com/dotnet/roslyn/compare/dd3d407264e187c84d4b3e5e53921394da6f5e4b..c7c1262dcd54fe9b0b11fe4d3dfd8cd9df986d08.

Reply:
@sharwell 

> Amended the commit to include a comment.

Please do not do this for PRs under review. Rewrites like this confuse CodeFlow tool that compiler team relies on for code reviews.

Reply:
Compiler team has a general policy that one pull request equals one commit in history, even though policies like that are problematic for tools (e.g. Git Extensions). I strive to make my pull requests match the expectations of the merge history.

I already make a limited exception for the case where you specifically have started a review, opting to rewrite the pull request at the end of the review instead of during it.